.make.defaults

```diff
@@ -270,28 +272,9 @@ __check_defined = \
 	@# Help: Build the Python $(DOCKER_LOCAL_IMAGE) using the the wheel file for the library
 	@$(eval LIB_WHEEL_FILE := $(shell find data-processing-dist/*.whl))
```

Comment:
Again,
`
$(call check_defined, DOCKER_LOCAL_IMAGE)`

Reply:
Not in this iteration. Most of this will be changing in a second iteration. For now, If it is not breaking the code, let's skip it. Thanks